resources/views/auth/passwords/reset.blade.php

```diff
@@ -15,12 +15,17 @@
                         <h4>{{ trans('auth.validation.issues') }}</h4>
                         <ul class="list -compacted">
                             @foreach ($errors->all() as $error)
-                                <li>{{ $error }}</li>
+                                @if (strpos($error, 'create-password'))
+                                <li>
+                                {{ $error }} 
+                                <a href="/password/reset">Get a new password link</a>
+                                </li>
+                                 @else <li>{{ $error }}</li>
+                                 @endif
```

Comment:
Since we're always wrapping this in an `<li>` and rendering the `$error` it could be neat to just wrap the link in the conditional:
```suggestion
                                <li>
                                    {{ $error }} 
                                    @if (strpos($error, 'create-password'))
                                        <a href="/password/reset">Get a new password link</a>
                                    @endif
                                </li>
```